arm/Microsoft.Network/applicationGateways/.parameters/parameters.json

```diff
@@ -3,159 +3,358 @@
     "contentVersion": "1.0.0.0",
     "parameters": {
         "name": {
-            "value": "<<namePrefix>>-az-apgw-x-001"
-        },
-        "sku": {
-            "value": "WAF_v2"
-        },
-        "vNetName": {
-            "value": "adp-<<namePrefix>>-az-vnet-x-001"
-        },
-        "subnetName": {
-            "value": "<<namePrefix>>-az-subnet-x-007"
-        },
-        "vNetResourceGroup": {
-            "value": "validation-rg"
-        },
-        "frontendPrivateIpAddress": {
-            "value": "10.0.8.6"
-        },
-        "frontendPublicIpResourceId": {
-            "value": "/subscriptions/<<subscriptionId>>/resourceGroups/validation-rg/providers/Microsoft.Network/publicIPAddresses/adp-<<namePrefix>>-az-pip-x-apgw"
+            "value": "sxx-az-apgw-x-001"
         },
         "userAssignedIdentities": {
             "value": {
-                "/subscriptions/<<subscriptionId>>/resourcegroups/validation-rg/providers/Microsoft.ManagedIdentity/userAssignedIdentities/adp-<<namePrefix>>-az-msi-x-001": {}
+                "/subscriptions/<<subscriptionId>>/resourcegroups/validation-rg/providers/Microsoft.ManagedIdentity/userAssignedIdentities/adp-sxx-az-msi-x-001": {}
             }
         },
-        "sslCertificateKeyVaultSecretId": {
-            "value": "https://adp-<<namePrefix>>-az-kv-x-001.vault.azure.net/secrets/applicationGatewaySslCertificate/9b670bb436e04eae9acd9b93dca269b6" // ID must be updated for new certificates
+        // "webApplicationFirewallConfiguration": {
+        //     "enabled": true,
+        //     "firewallMode": "Detection",
+        //     "ruleSetType": "OWASP",
+        //     "ruleSetVersion": "3.0",
+        //     "disabledRuleGroups": [],
+        //     "requestBodyCheck": true,
+        //     "maxRequestBodySizeInKb": 128,
+        //     "fileUploadLimitInMb": 100
+        // },
+        // "enableHttp2": true,
```

Comment:
Is this to be removed?

Reply:
Unfortunately not. It is only sometimes working. Need to iterate on that

Reply:
fixed it